src/common/helpers/ImportHelpers.ts

```diff
@@ -13,6 +15,6 @@ export function getImportInsertPosition(editor: TextEditor | undefined): Positio
         return new Position(0, 0);
     }
     const lines = editor.document.getText().split('\n');
-    const index = lines.findIndex(line => !line.match(/^\s*(\/\/|\/\*\*|\*\/|\*|['"]use strict['"])/g));
+    const index = lines.findIndex(line => !REGEX_IGNORED_LINE.test(line));
```

Comment:
- `regex.test(x)` perfs better than `x.match(regex)` when all we need is a yes/no